doc/man/man3/crypto_lock.3monocypher

```diff
@@ -203,12 +204,12 @@ crypto_wipe(key, 32);
 .Pp
 To decrypt the above:
 .Bd -literal -offset indent
-const uint8_t  key  [32];    /* Same as the above         */
-const uint8_t  nonce[24];    /* Same as the above         */
-const uint8_t  mac  [16];    /* Received from the network */
-const uint8_t *cipher_text;  /* Encrypted message         */
-size_t         text_size;    /* Message size (NOT secret) */
-uint8_t       *plain_text;   /* Secret message            */
+const uint8_t  key  [32];   /* Same as the above         */
+const uint8_t  nonce[24];   /* Same as the above         */
+const uint8_t  mac  [16];   /* Received from the network */
```

Comment:
is "Received from the network" right here?

Reply:
Strictly speaking, any unsecure channel would do, it doesn't have to be the network specifically. The network is just a typical example. I don't know a concise way to say it, though. "Received from the sender", maybe? "Sent by the sender"? I think "Received from the network" is good enough, but we can change it if you prefer something else.

Reply:
I don't mind it but you guys were talking about it on one of the other PRs

Reply:
Bah, I'd rather leave it alone it looks good enough.